target/linux/bcm53xx/image/Makefile

```diff
@@ -170,21 +174,24 @@ endef
 
 define Device/buffalo-wxr-1900dhp
   $(call Device/buffalo/Default)
```

Comment:
Sorry, I overlooked the common definitions when I reviewed the first time.
Can this be exploited for common DEVICE_VENDOR? (Or is there something specific to "call" that prevents it)